media/css/privacy/cookie-settings-form.scss

```diff
@@ -80,6 +80,11 @@ main {
 
     .cookie-summary {
         border-bottom: 1px solid $color-light-gray-40;
+
+        // Temporary fix until https://github.com/mozilla/protocol/issues/948 is added
+        &:hover {
```

Comment:
Not a blocker, but I don't think `&:hover` isn't really necessary since it performs the same action